pal/samples/esp8266/iothub_ll_telemetry_sample.ino

```diff
@@ -129,15 +132,15 @@ void setup() {
 
     sample_init(ssid, pass);
 
+    // Create the iothub handle here
     device_ll_handle = IoTHubDeviceClient_LL_CreateFromConnectionString(connectionString, protocol);
     // Used to initialize IoTHub SDK subsystem
     (void)IoTHub_Init();
-
+    
     LogInfo("Creating IoTHub Device handle\r\n");
-    // Create the iothub handle here
     if (device_ll_handle == NULL)
     {
-        LogInfo("Failure createing Iothub device.  Hint: Check you connection string.\r\n");
+        LogInfo("Failure createing Iothub device. AZ002 Hint: Check you connection string.\r\n");
```

Comment:
what is `AZ002`

Reply:
It's added for the python test scraper, could probably change it to Error AZ002: Failure to ...